gazelle/pythonconfig/pythonconfig_test.go

```diff
@@ -0,0 +1,26 @@
+package pythonconfig
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/bazelbuild/rules_python/gazelle/pythonconfig"
+)
+
+func TestDistributionSanitizingUpperCase(t *testing.T) {
+	distname := "DistWithUpperCase"
+	sanitized := pythonconfig.SanitizeDistribution(distname)
+
+	if sanitized != strings.ToLower(distname) {
```

Comment:
I'd have expected a list of probable inputs and the expected outputs. Dave Cheney wrote a good article about it here: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests.

Reply:
For sure, I'm not that familiar with Go. But more than happy to rewrite following best practices.